src/anyio/_backends/_asyncio.py

```diff
@@ -372,11 +372,22 @@ def _task_started(task: asyncio.Task) -> bool:
 
 
 def is_anyio_cancellation(exc: CancelledError) -> bool:
-    return (
-        bool(exc.args)
-        and isinstance(exc.args[0], str)
-        and exc.args[0].startswith("Cancelled by cancel scope ")
-    )
+    # Sometimes third party frameworks catch a CancelledError and raise a new one, so as
```

Comment:
Can you include this in the change log?

Reply:
Oh I see oh well

Reply:
Let's do (or don't) the warning on another PR. I'll investigate asyncpg

Reply:
looks like it's fixed in asyncpg: https://github.com/MagicStack/asyncpg/blob/e9bb695413b2ee923310f9106277198cc19c47b3/asyncpg/_asyncio_compat.py#L25-L73